src/utils/telemetry.ts

```diff
@@ -47,35 +46,3 @@ function sendTelemetry(eventName: string, properties?: { [key: string]: string;
         reporter.sendTelemetryEvent(eventName, properties, measures);
```

Comment:
I don't see the function `sendTelemetry` being used, now that the `callWithTelemetry` function is removed.